src/dataProvider.ts

```diff
@@ -121,6 +121,9 @@ const dataProviderWithCustomMethods = {
             throw new Error('Failed to create account');
         }
 
+        // Update the is initialized cache
+        getIsInitialized._is_initialized_cache = true;
```

Comment:
You're storing it in memory, which isn't enough. In the use case of a user opening their browser for the first time of the day, but using a valid token in localStorage, they should also skip the initialization check.

Reply:
I'm not sure that this is a good idea to perist it in the local storage: if a developer restets its DB, the web app will consider the app to be initialized, which is not the case